apps/cli-daemon/src/app/workspace/workspace.controller.ts

```diff
@@ -34,6 +36,17 @@ export class WorkspaceController {
     this.sessionService.setCwd(path);
   }
 
+  @Post('create')
+  create(@Body() body: WorkspaceCreateDto): ExecResult {
+    try {
+      this.sessionService.setCwd(body.path);
+      return this.workspaceService.execSync(this.ngNewArgsFromDto(body));
```

Comment:
This won't work well. For the workspace generation, we need an async executor and not this one. I planned to create one when we'll decide if want executors for `serve` and `build`. Didn't know that you are going to start with `ng new`.

Reply:
Why won't it work well? How's it different than generating anything else?